Setup/DownloadExternals.cmd

```diff
@@ -2,8 +2,8 @@
 
 REM Also update in Product.wxs and UI\RequiredSoftwareDlg.wxs
 
-IF NOT EXIST "%~p0\..\bin\Git-2.10.1-32-bit.exe" (
-    "%~p0\tools\curl.exe" -L -k -o ..\bin\Git-2.10.1-32-bit.exe https://github.com/git-for-windows/git/releases/download/v2.10.1.windows.1/Git-2.10.1-32-bit.exe
+IF NOT EXIST "%~p0\..\bin\Git-2.12.2-64-bit.exe" (
```

Comment:
> perhaps check the architecture first and install 32- or 64-bit where appropriate
> few ways to check bitness

If we do so, installer will include both 32- and 64-bit Git installers.
I think better just use 32-bit installer